components/ws-manager/pkg/manager/monitor.go

```diff
@@ -1310,6 +1312,14 @@ func (m *Monitor) finalizeWorkspaceContent(ctx context.Context, wso *workspaceOb
 		if (err != nil && strings.Contains(err.Error(), context.DeadlineExceeded.Error())) ||
 			st.Code() == codes.Unavailable ||
 			st.Code() == codes.Canceled {
+			if disposalStatus.Status != DisposalRetrying {
+				disposalStatus.Status = DisposalRetrying
```

Comment:
Probably we don't need the `retrying` state, because in general, the behavior is the same as `started` state.

Reply:
Many errors have occurred most recently in the monitor. I believe that any information that can give us some clues should be given. In the case of retrying, at least once the finalizer makes it clear that it has failed. WDYT?

Reply:
Does the rest of the code work properly if it will encounter status `DisposalRetrying`? If yes, then it is probably a good idea to keep it.

Reply:
I want to double confirm that do we need to add one more status? But looks like it will give us more information on this. So, I am okay to add the retrying status 💪